packages/components/src/tools-panel/tools-panel-header/component.tsx

```diff
@@ -118,7 +135,7 @@ const ToolsPanelHeader = ( props, forwardedRef ) => {
 					label={ labelText }
 					menuProps={ { className: dropdownMenuClassName } }
 				>
-					{ ( { onClose } ) => (
+					{ ( { onClose = () => undefined } ) => (
```

Comment:
I would normally prefer not to assign a default value here, but since in this case `onClose` would come from an un-typed component (`Dropdown`), we can leave it here.

To avoid creating a new function every time, we should probably define something like `const noop = () => {};` at the very top of the file (immediately after the imports), and then use it here:

```suggestion
					{ ( { onClose = noop } ) => (
```